### ISSUE TITLE
Installation docs for 2.0 inconsistant w.r.t drbd's minor_count option

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#132]
Originally reported of Google Code with ID 76.

```
http://ganeti-doc.googlecode.com/svn/ganeti-2.0/install.html

Under "Installing DRBD", 4th paragraph, it says to set minor_count=255, but
all the other text and the example code has minor_count=128.  I assume this
is a typo?
```
Originally added on 2009-10-23 22:55:37 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Not a typo, just an inconsistency - the text says "set to 255, but for most clusters
128 is enough". I'm in process of updating the docs so I'll take care of this.

thanks,
iustin
```
Originally added on 2009-10-24 05:50:55 +0000 UTC.

Added Labels: Component-docs
Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
This is fixed in the 2.1 documentation. Marking as closed.

regards,
iustin
```
Originally added on 2010-07-28 23:26:01 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
